octave/tm/tmrepl.m

```diff
@@ -29,6 +33,7 @@ function tmrepl()
       # Reset ans to empty string
       ans= "";
       # Suppress the output
+      __r = strrep(__r,"\n",";\n");
```

Comment:
This line changes the syntax of the octave script, and it is not necessary.

Reply:
The global ans will be displayed after the block of code executed. For one line or multi lines, only one ans will be displayed.

Reply:
Agreed, I have undone this change.